README-short.txt

```diff
@@ -1 +1 @@
-Crate is a shared nothing, fully searchable, SQL database with the NoSQL goodies.
+Crate - the fast, scalable, easy to use SQL database.
```

Comment:
Crate is an open source, highly scalable, shared-nothing distributed SQL database.